source/code-snippets/usage-examples/find.js

```diff
@@ -0,0 +1,28 @@
+
+const { MongoClient } = require('mongodb');
+
+
+const client = new MongoClient(
+   'mongodb+srv://admin:admin@mflix-2sp0m.mongodb.net/test?retryWrites=true&w=majority',
+);
+
+async function run() {
+   try {
+      await client.connect();
+
+      const database = client.db('sample_mflix');
+      const collection = database.collection('movies');
+
+      // create a query document for a range match for a movie less than 15 minutes long
+      const query = { runtime: { $lt : 15 } };
+      const cursor = collection.find(query);
```

Comment:
should this be: `await collection.find(query);`

Reply:
It works either way -- I suspect since the cursor is lazy, the fact that we use an await below makes it unnecessary. I'll add an await here though because I suspect we're "better safe than sorry" when it comes to new users who might play around with/edit these examples to better understand what's going on (and then use the resulting code in personal projects.